components/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
```

Comment:
You should remove this file too because it's only used by your text editor

Reply:
Got it, it was removed :smile: